test/distributed/org/apache/cassandra/fuzz/sai/SingleNodeSAITestBase.java

```diff
@@ -104,28 +111,49 @@ public static void afterClass()
     @Before
     public void beforeEach()
     {
-        cluster.schemaChange("DROP KEYSPACE IF EXISTS harry");
-        cluster.schemaChange("CREATE KEYSPACE harry WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};");
+        cluster.schemaChange("DROP KEYSPACE IF EXISTS " + KEYSPACE);
+        cluster.schemaChange("CREATE KEYSPACE " + KEYSPACE + " WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};");
```

Comment:
feel free to ignore for this PR, but wouldn't it be easier to maintain this logic with regard to multi node if `rf()` was a function?